man/systemd.netdev.xml

```diff
@@ -911,6 +916,23 @@
           <para>Accepts the same key as <literal>[FooOverUDP]</literal></para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Sequence=</varname></term>
+        <listitem>
+          <para> The <varname>Sequence</varname> Serializes the packets. Only applies for ERSPAN tunnel.
```

Comment:
leading space is weird, and "Serializes" should be lowercase. Also, this should say what kind of values this takes? strings? integers? bools? also, what's the default?